decision_path_with_concept.py

```diff
@@ -0,0 +1,114 @@
+from config import dataset
+from utils.graph.framework import display_decision_tree
+from utils.graph.graph_data import get_cluster_previews, image_segment
+from utils.graph.nodes import edge_weight, class_node, edge, cluster_node, class_node_id, cluster_node_id, segment_node_id, segment_node, class_node_image_src
+from steps.s1_build_segments import load_segments_of, segment_an_image
+from steps.s2_interpret_segments import interpret_an_image
+from steps.s5_build_decision_tree import predict_bird_by_id, predict_bird_by_activation
+target_bird_id = 11750 #11700 #11000 #10000# 9000 #3000 #900 #1 #868, #1000
+
+
+def has_concept(weight):
+    if weight:
+        weight = '✅'
+    else:
+        weight = '❌'
+    return weight
+
+def draw_decision_path(image_id, dataset):
```

Comment:
Is this even needed at all? It looks like nothing from this file is actually used and showing similar birds is also implemented in `decistion_path.py`?